extended/jams2015.tex

```diff
@@ -21,7 +21,7 @@
 \colorlet{numb}{magenta!60!black}
 
 \lstdefinelanguage{json}{
-    basicstyle=\normalfont\ttfamily,
+    basicstyle=\normalfont\ttfamily\scriptsize,
```

Comment:
But that should just be in the two example figures.  Changing it here in the lst definition makes all of the code listings tiny.